home.css

```diff
@@ -0,0 +1,720 @@
+}
+
+.m1 > h3 {
+  padding-right: 60px;
+}
+
+.m2 > h3 {
+  padding-right: 40px;
+}
+
+.m3 > h3 {
+  padding-right: 60px;
+}
+
+.m4 > h3 {
+  padding-right: 40px;
+}
+
+.m5 > h3 {
+  padding-right: 60px;
+}
+
+.spkr2 h3 {
+  font-weight: 700;
+  font-size: 15px;
+  line-height: 18px;
+  letter-spacing: -0.02em;
+  color: #000;
+}
+
+.main-content > ul > li > h3 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14.2712px;
+  line-height: 25px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.01em;
+  color: #ff5438;
+  padding-left: 10px;
+}
+
+.spkr2 p {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 9px;
+  line-height: 14px;
+  letter-spacing: -0.06em;
+  color: #000;
+}
+
+.main-content > ul > li > p {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 10.7034px;
+  line-height: 22px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.06em;
+  color: #fff;
+}
+
+.m1 {
+  margin-bottom: 3px;
+}
+
+li.m2 {
+  margin-bottom: 3px;
+}
+
+li.m3 {
+  margin-bottom: 3px;
+}
+
+li.m4 {
+  margin-bottom: 3px;
+}
+
+li.m5 {
+  margin-bottom: 3px;
+}
+
+.featured > h2 {
+  text-align: center;
+  margin: 0;
+}
+
+.joinbtn button {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 20px 50px;
+  gap: 10px;
+  width: 90%;
+  height: 68px;
+  background: #ff4f38;
+  margin-right: 40px;
+  margin-left: 20px;
+  border: 0;
+}
+
+.spkr {
+  list-style: none;
+  padding: 0;
+}
+
+.spkr1 {
+  list-style: none;
+  display: flex;
+}
+
+.spkr2 {
+  list-style: none;
+}
+
+.spkr2 h4 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 10px;
+  line-height: 15px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.02em;
+  color: #f78c89;
+  margin: 0;
+}
+
+.spkr2 hr {
+  width: 17.79px;
+  height: 0;
+  border: 1.42315px solid #d9d9d9;
+  margin: 0;
+  margin-bottom: 6px;
+}
+
+body > section > section.featured > div:nth-child(2) {
+  margin-top: 20px;
+}
+
+.loadMore {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 32px;
+}
+
+.loadMore > button {
+  width: 90%;
+  border: 1px solid #dcdcdc;
+  box-shadow: 1px 1px 1px 1px lightgray;
+  border-radius: 4px;
+}
+
+.desktop {
+  display: none;
+}
+
+.footer {
+  display: flex;
+  margin: 40px 18px;
+  gap: 15%;
+}
+
+.footer-des {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 10px;
+  line-height: 13px;
+  letter-spacing: -0.01em;
+  color: #3b3e45;
+  width: 50%;
+  margin: 0;
+}
+
+#footterkor {
+  display: none;
+}
+
+#partner-des {
+  display: none;
+}
+
+#visibile {
+  display: flex;
+}
+
+#hidden {
+  display: none;
+}
+
+@media only screen and (max-width: 768px) {
+  .showcontent {
+    background-color: #3b3e45;
+    background-size: cover;
+    color: #fff;
+    background-repeat: no-repeat;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    mix-blend-mode: multiply;
+    display: flex;
```

Comment:
[ Optional ] I think it would be better if the mobile menu is hidden by default using the CSS `display: none` property. Currently, when I switch to the mobile version the `mobile menu` is displayed by default even before I click on the hamburger menu. Using `display: none` will hide the menu by default such that the only way to open the mobile menu will be by clicking on the hamburger menu.